docs/pages/web/button.js

```diff
@@ -624,80 +543,36 @@ Used to block user interaction such as hover, focus and click. Disabled Buttons
         >
           <MainSection.Card
             cardSize="md"
-            defaultCode={`
-<Button
-  accessibilityLabel='Save'
-  color="red"
-  text="Save"
-  size="lg"
-/>
-`}
+            sandpackExample={
```

Comment:
For these three states we could implement a preview height that takes less height, no?

Reply:
This is happening because this line: https://github.com/pinterest/gestalt/blob/master/docs/docs-components/SandpackExample.js#L49